config/FleCSIConfig.cmake.in

```diff
@@ -27,6 +27,8 @@ endif()
 #------------------------------------------------------------------------------#
 
 set(FleCSI_LIBRARY FleCSI)
-set(FleCSI_LIBRARIES ${FLECSI_LIBRARY_DEPENDENCIES})
+# Normal cmake convention includes the main library plus all of its
+# dependancies in <LIBRARY>_LIBRARIES
+set(FleCSI_LIBRARIES ${FLECSI_LIBRARY_DEPENDENCIES} ${FleCSI_LIBRARY})
```

Comment:
Does ordering make a difference here?  If this variable is used directly as-is in a link command, FleCSI_LIBRARY would need to come first.